config/samples/mysql_v1alpha1_mysqlcluster.yaml

```diff
@@ -4,7 +4,6 @@ metadata:
   name: sample
 spec:
   replicas: 3
-  mysqlVersion: "5.7"
```

Comment:
becuase it is a change of the usage habit， should add some comments， and point out from which version it has，and which version this field is disappear.

Reply:
Yes ,we can keep this value for Compatibility Considerations

Reply:
done